api/src/pcapi/routes/backoffice/admin/blueprint.py

```diff
@@ -129,3 +130,13 @@ def toggle_feature_flag(feature_flag_id: int, set_to_active: bool) -> utils.Back
         f"Le feature flag {feature_flag.name} a été " + ("activé" if feature_flag.isActive else "désactivé"), "success"
     )
     return redirect(url_for(".list_feature_flags"), code=303)
+
+
+@blueprint.backoffice_web.route("/admin/category-subcategory", methods=["GET"])
+@atomic()
+@utils.permission_required(perm_models.Permissions.CATEGORY_AND_SUBCATEGORY)
+def get_category_subcategory() -> utils.BackofficeResponse:
+
+    all_subcategory = subcategories_v2.ALL_SUBCATEGORIES_DICT.values()
```

Comment:
Si elles y sont toutes, je pense qu'il y en aura plusieurs :
```suggestion
    all_subcategories = subcategories_v2.ALL_SUBCATEGORIES_DICT.values()
```

Reply:
`    all_subcategories = subcategories_v2.ALL_SUBCATEGORIES_DICT.values()`
FAIT